interface-definitions/include/qos/queue-maximum-threshold.xml.i

```diff
@@ -0,0 +1,16 @@
+<!-- include start from qos/queue-maximum-threshold.xml.i -->
+<leafNode name="maximum-threshold">
+  <properties>
+    <help>Maximum threshold for random detection</help>
+    <valueHelp>
+      <format>u32:0-4096</format>
+      <description>Maximum Threshold in packets</description>
```

Comment:
Nitpick: any specific reason to capitalize "threshold" here?
I'd also say "minimum number of packets" here in the value help.

Reply:
I just got this from the old version as is, but I agree, we don't have any other reason to capitalize this.
https://github.com/vyos/vyos-1x/blob/6d8336f5ad2d9c4e0f12b54681db2924d6998d2d/interface-definitions/qos.xml.in#L506
About "minimum number of packets" looks like "maximum threshold" is a common term when describing this protocol, and probably better to use threshold instead of number in our description

Reply:
capitalize fixed